packages/node/src/handlers/test-api.ts

```diff
@@ -0,0 +1,57 @@
+import find from 'lodash/find';
+import * as wallet from '../evm/wallet';
+import { randomString } from '../utils/string-utils';
+import { AggregatedApiCall, Config, WorkerOptions, ApiCallResponse } from '../types';
+import * as logger from '../logger';
+import { go } from '../utils/promise-utils';
+import { spawnNewApiCall } from '../adapters/http/worker';
+import { WORKER_CALL_API_TIMEOUT } from '../constants';
+
+export async function testApi(
+  config: Config,
+  endpointName: string,
+  parameters: Record<string, string>
+): Promise<[Error, null] | [null, ApiCallResponse]> {
+  const testCallId = randomString(8);
+  const masterHDNode = wallet.getMasterHDNode();
+
+  const logOptions = logger.buildBaseOptions(config, { requestId: testCallId });
+
+  const triggerRequest = find(config.triggers.request, ['endpointName', endpointName]);
+  if (!triggerRequest) {
+    return [new Error(`No such endpoint '${endpointName}'`), null];
+  }
+
+  const workerOpts: WorkerOptions = {
+    cloudProvider: config.nodeSettings.cloudProvider,
+    airnodeIdShort: wallet.getAirnodeIdShort(masterHDNode),
+    stage: config.nodeSettings.stage,
+    region: config.nodeSettings.region,
+  };
+
+  const aggregatedApiCall: AggregatedApiCall = {
+    id: testCallId,
+    airnodeId: wallet.getAirnodeId(masterHDNode),
+    requesterIndex: '',
+    clientAddress: '',
+    designatedWallet: '',
+    chainId: '',
+    endpointId: triggerRequest.endpointId,
+    endpointName,
+    oisTitle: triggerRequest.oisTitle,
+    parameters,
+  };
+
+  const [err, logData] = await go(() => spawnNewApiCall(aggregatedApiCall, logOptions, workerOpts, false), {
+    timeoutMs: WORKER_CALL_API_TIMEOUT,
```

Comment:
See comment about timeouts above. This would time out after 30 seconds but the lambda has a timeout of 60 seconds (currently)